main.py

```diff
@@ -19,6 +19,89 @@
 def hello():
     return 'Hello, Circles!!'
 
+# User API
+
+
+@app.route('/circles/api/v1.0/users/<id>', methods=['GET'])
+def get_person(id):
+    return """{"person": {
+        "id": "123",
+		"display_name": "Cool Dude", 
+		"email": "cooldude@duke.edu:,
+		"photo": "base64",
+		"Knows": [],
+		"IsMember": [],
+		"InvitedTo": []
+    }
+}"""
+
+
+# @app.route('circles/api/v1.0/users/<int:user_id>', methods=['POST'])
```

Comment:
I believe for Flask API's you generally would group two of the same URI's under the same handler method. So instead of having two different handlers, just have a single one and then method=['GET', 'POST'] but look into this a bit.

Reply:
I guess if we are going to submit these stubs as separate PR, we don't have to do this now.